spec/cypress/integration/example-tests.spec.js

```diff
@@ -0,0 +1,125 @@
+describe("Example tests", () => {
+  beforeEach(() => {
+    cy.app("clean");
+  });
+
+  after(() => {
+    cy.app("clean");
+  });
+
+  // This is an example of a failing test - skipping because it takes a while to run
+  it.skip("should be viewable when logged out", () => {
+    cy.appFactories([["create", "course_lesson"]]);
+
+    cy.visit("/");
+
+    cy.get("#navigation").contains("Core Induction Programme").click();
+
+    cy.get("h1").should("contain", "What is the early career framework?");
+
+    cy.contains("Test Course year").click();
+
+    cy.get("h1").should("contain", "Test Course year");
+    cy.get(".govuk-govspeak").should("contain", "No content");
+
+    cy.get("a").contains("Test Course module").click();
+
+    cy.get("h1").should("contain", "Test Course module");
+    cy.get(".govuk-govspeak").should("contain", "No content");
+
+    cy.get(".app-task-list .govuk-tag").should("not.exist");
+
+    // Above test will fail for demo so not much point continuing…
+    cy.get("a").contains("Test Course lesson").click();
+  });
+
+  it("should not be editable when logged in as non-admin user", () => {
+    cy.appFactories([
+      ["create", "user"],
+      ["create", "course_lesson"],
+    ]);
+
+    // @todo appFactories returns this
+    cy.appEval("User.last").then((res) => {
```

Comment:
When fetching something we just created, we should include an `.order(:created_at)` before the last